cmd/kubelet/kubelet.go

```diff
@@ -34,13 +34,24 @@ import (
 )
 
 func main() {
+	err := runKubeletCmd()
+	defer func() {
```

Comment:
I think we don't have to use defer here.
- whenever runKubeletCmd() returns error, we want to log and exit(1)
- when runKubeletCmd() panic, we can see stack trace (as you said ([ref](https://github.com/kubernetes/kubernetes/pull/104776#discussion_r715827949))) and I agree with your opinion -- we don't have to recover and print it, I think.

Are there any cases where this is preferable?

Reply:
since no recover() will called, I'll refactor this code. the same with others.